### ISSUE TITLE
New French OPV

### ISSUE THREAD

**Issue by gosnold** [PygmalionOfCyprus/cmo-db-requests#4013]
### DB Selector

DB3K

### Hypothetical

No

### Name

patrouilleur hauturier

### Class

patrouilleur hauturier

### Country

France

### Service

Navy

### In Commission

2026

### Number in Service

1x 2026, 7x 2030, 10x 2035

### Length (m)

92

### Beam (m)

?

### Draft (m)

?

### Standard Displacement (t)

2400

### Complement

50+34

### Primary Propulsion Type

Diesel

### Propulsion

?

### Flank Speed (kt)

21

### Range (nm) at Speed (kt)

6000nm

### Sensors

1x NS-50 radar from Thales (surface and air search, 180km max range)
1x IFF
1x Rapidfire optical fire director
1x Safran Vigy optronics
1x Thales Bluewatcher Sonar
1x navigation radar
Maybe generic ESM?

### Mounts

1x Rapidfire 40mm mount  (front, ~270° arc)
1x Small arms on each side

### Aircraft & Docking Facilities

1x pad and hanger for 6t helo (H160M and/or VSR700 drone)
2x 9m RHIB davits
1x SMDM catapult + net recovery drone

### Comms/Datalinks

- Link 22
- SATCOM (syracuse 4 and maybe commerical)

### Sources

https://www.navalnews.com/naval-news/2023/11/frances-dga-orders-7-new-opv-for-the-french-navy/

https://www.naval-group.com/fr/naval-group-acheve-la-conception-des-futurs-patrouilleurs-hauturiers-de-la-marine-nationale-et

https://www.defense.gouv.fr/dga/actualites/dga-commande-sept-premiers-batiments-du-programme-patrouilleurs-hauturiers

https://lemarin.ouest-france.fr/industries-navales/les-futurs-patrouilleurs-hauturiers-apporteront-de-gros-progres-capacitaires-ff99f790-8a22-11ee-a92a-0b4bc05f91ae

https://www.thalesgroup.com/en/worldwide/defence/press_release/thales-selected-dga-equip-french-navys-new-offshore-patrol-vessels

**Comment by claudejdev** [comment]
Just a quick note: "patrouilleur hauturier" = OPV
https://www.navyrecognition.com/index.php/naval-news/naval-news-archive/2023/november/13819-france-orders-7-corvettes-reviving-2nd-part-of-batsimar-program.html

**Comment by gosnold** [comment]
Yeah I put that for the name of the class but it's the name of the program, we don't know the name of the first ship yet.
